crates/sargon/src/wrapped_radix_engine_toolkit/high_level/ret_api.rs

```diff
@@ -142,14 +142,6 @@ pub fn xrd_address_of_network(network_id: NetworkID) -> ResourceAddress {
     ResourceAddress::xrd_on_network(network_id)
 }
 
-#[uniffi::export]
-pub fn debug_print_compiled_notarized_intent(
-    compiled: CompiledNotarizedIntent,
-) -> String {
-    let notarized = compiled.decompile();
-    format!("{:?}", notarized)
```

Comment:
This isn't used on Android, and on iOS we weren't [printing anything](https://github.com/radixdlt/babylon-wallet-ios/blob/main/RadixWallet/Clients/SubmitTransactionClient/SubmitTransactionClient%2BLive.swift#L58), just declaring a String that Sargon provided (but not logging it), so dediced to remove.